### PR TITLE
Pokémon full view

### PR DESCRIPTION
**Pull request by NotRaia**
This isn't anywhere near ready, but I just wanted to make a task list:

- [x] Create page skeleton
- [x] Fill in page with info from pk6
- [x] Is it necessary to have `pokemon.fetch()` for every card? If not, let's fix it
- [ ] Get type data
- [x] Get forme data
- [ ] Get base/total stats
- [ ] Get memory text
- [ ] Get female sprite when applicable
- [ ] Create new views for each tab to avoid overloading one page
- [x] Rename Pokémon images to IDs
- [ ] Rename item images to IDs
- [x] Fix design, make responsive

Mockups for reference: http://imgur.com/a/hO0gs

### CODE REVIEW

**Comment by not-an-aardvark** [comment]
> Is it necessary to have pokemon.fetch() for every card? If not, let's fix it

It looks like at the moment, the client is re-fetching the pokemon from the server for each individual piece of info, which probably isn't what we want. (You can see when a request gets made by looking at the logged info in the terminal for the local server.)

I think if you put the `ng-init=pokemon.fetch()` tag in an outer scope, you won't need to use it more than once.

**Comment by NotRaia** [comment]
>Rename and implement large sprites (probably best to follow pokesprite nomenclature)

A note on this: It might not be too bad doing the non-shiny versions manually and just writing a script for the shiny ones (as they are probably in the same order). Trying to write a script for *everything* and ending up with undesired results may just waste more time.

**Comment by YaManicKill** [comment]
> I think if you put the ng-init=pokemon.fetch() tag in an outer scope, you won't need to use it more than once.

Or if you put it in a controller.

**Comment by not-an-aardvark** [comment]
The issue is that we currently use the same controller for cards and the full view, so if we directly put `pokemon.fetch()` in a controller, then the pokemon will be fetched whenever someone views a card, which isn't what we want.

**Comment by YaManicKill** [comment]
Then either

1. Use a different controller that extends the existing one
2. Use 2 controllers

File: client/pokemon/pokemon-full.view.html [hunk 1]
```diff
@@ -1,12 +1,315 @@
...
+          <md-card class="summary" ng-show="pokemon.hasFullData">
+            <md-card-header><span class="md-headline">Upload Info</span></md-card-header>
+            <md-card-content>
+              <!-- add link to owner's profile -->
+              <div><strong>Owner</strong> {{pokemon.data.owner}}</div>
+              <div><strong>Unique in Porybox database?</strong>
+                <span ng-if="pokemon.data.isUnique">Yes</span>
+                <span ng-if="!pokemon.data.isUnique">No</span>
+              </div>
+              <div><a ng-href="/#/box/{{pokemon.data.box}}">Back to box</a></div>
```
Comment (not-an-aardvark) [inline client/pokemon/pokemon-full.view.html]: Be sure to account for the case where the viewer is not the pokemon's owner; `pokemon.data.box` will be undefined.

**Comment by not-an-aardvark** [comment]
We decided to merge now and fix the rest of the issues afterwards.

**Pull request merged**